source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -254,10 +254,10 @@ was generated using SpongeForge build 1399, SpongeAPI version 4.1:
             chunk-gc-load-threshold=0
             # The tick interval used to cleanup all inactive chunks in a world. 
             # Set to 0 to disable which restores vanilla handling. (Default: 1)
```

Comment:
That's what we usually do (although the entity lists self-populate as they are encountered so they can be a pain). We'll generate one when it appears in an implementation.